packages/harpguru-core/src/styles/sizescheme/use-sizes.ts

```diff
@@ -0,0 +1,135 @@
+import { useGlobal } from 'reactn'
+
+import { SizeScheme, SizeSchemes } from '../styles-types'
+import { getWindowDimensions } from '../../packages/get-window-dimensions'
+
+// Here's what we need this hook to provide:
```

Comment:
This plan can be removed now

Reply:
Raised as techdebt here: https://github.com/js-jslog/harpguru/issues/113#issuecomment-922036125

The work done to make the app sizing dynamic (at #107) was used as a platform for additional speculative pull requests to ensure that this was the correct direction to start down.

Rather than addressing the finer details in this or subsequent pull request and potentially struggling to merge up to those subsequent branches I will prefer to record the issues in each merge request and resolve them afterwards.

This has a limit of course. Major issues will not be merged to master. I will need to find a way around that if it happens.